packages/editor/src/client/blocks/teaserFlexGridBlock.tsx

```diff
@@ -0,0 +1,278 @@
+import React, {useEffect, useState} from 'react'
+import GridLayout from 'react-grid-layout'
+
+import 'react-grid-layout/css/styles.css'
+import 'react-resizable/css/styles.css'
+
+import {FlexItemAlignment, FlexTeaser, Teaser, TeaserFlexGridBlockValue} from './types'
+import {BlockProps} from '../atoms/blockList'
+import nanoid from 'nanoid'
+import {ButtonToolbar, Drawer, Icon, IconButton, Panel} from 'rsuite'
+import {IconButtonTooltip} from '../atoms/iconButtonTooltip'
+import {contentForTeaser, TeaserBlockProps} from './teaserGridBlock'
+import {PlaceholderInput} from '../atoms/placeholderInput'
+import {TeaserEditPanel} from '../panel/teaserEditPanel'
+import {TeaserSelectAndEditPanel} from '../panel/teaserSelectAndEditPanel'
+
+// import {useTranslation} from 'react-i18next'
+
+export function FlexTeaserBlock({
+  teaser,
+  showGrabCursor,
+  onEdit,
+  onChoose,
+  onRemove
+}: TeaserBlockProps) {
+  return (
+    <Panel
+      bodyFill={true}
+      style={{
+        cursor: showGrabCursor ? 'grab' : '',
+        height: 'inherit',
+        overflow: 'hidden',
+        zIndex: 1
+      }}>
+      <PlaceholderInput onAddClick={onChoose}>
+        {teaser && (
+          <div
+            style={{
+              position: 'relative',
+              width: '100%',
+              height: '100%'
+            }}>
+            {contentForTeaser(teaser)}
+
+            <div
+              style={{
+                position: 'absolute',
+                zIndex: 1,
+                right: 0,
+                top: 0
+              }}>
+              {/* eslint-disable-next-line i18next/no-literal-string */}
+              <IconButtonTooltip caption="choose teaser">
```

Comment:
Don't we have these strings in translation files already? 👀 if not I would recommend adding them, what do you think?

Reply:
Yes. That'll be moved to translation later.